### PR TITLE
Adds examples as an embedded iFrame

### PR DESCRIPTION
**Pull request by dgasmith**
As discussed, pretty simple!

### CODE REVIEW

**Comment by dgasmith** [comment]
@mattwelborn Please review!

File: config.toml [hunk 1]
```diff
@@ -88,12 +90,27 @@ paginate = 6
...
   weight = 10
+  pre = "<u>"
+  post = "</u>"
+
+
+  # footer menu
+  [[menu.footer]]
+  name = "About"
+  URL = "/about"
+  weight = 1
```
Comment (mattwelborn) [inline config.toml]: Did you intend to add the About page back into the footer?

File: content/examples/_index.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+title: "QCArchive Examples"
+date: 2019-04-17T12:35:54+06:00
+description: "Stories and news about the QCArchive"
```
Comment (mattwelborn) [inline content/examples/_index.md]: description and pageDescription need to be updated (or maybe deleted?)

File: config.toml [hunk 3]
```diff
@@ -88,12 +90,27 @@ paginate = 6
...
+  weight = 9
+  name = "Examples"
+  URL = "/examples"
   
 
   [[menu.main]]
-  name = "QCA Infrastructure"
-  URL = "/projects"
+  name = "Infrastructure"
+  URL = "https://docs.qcarchive.molssi.org"
```
Comment (mattwelborn) [inline config.toml]: This will be the landing page for the infrastructure? Out of curiosity, what repo will it live in?

Comment (dgasmith) [reply config.toml]: One more repo ;/

https://github.com/MolSSI/QCArchiveInfrastructureWebsite

File: config.toml (hunk 1 above)
Comment (dgasmith) [reply config.toml]: I did not, fixed.

File: content/examples/_index.md (hunk 2 above)
Comment (dgasmith) [reply content/examples/_index.md]: Isn't used, but updated anyway.

**Comment by dgasmith** [comment]
@mattwelborn I snuck in the "Power by" at the bottom here too!

**Comment by dgasmith** [comment]
I also remembered that we must include the `_gen` folder because not everyone can reform scss.

**Pull request merged**